src/components/Plate/Plate.tsx

```diff
@@ -1,67 +1,101 @@
 import { classNames } from "@utils/stringUtils";
 import { SelectableGroup, createSelectable } from "react-selectable";
-import { RowAnnotation } from "@Ariadne/types";
 import { z } from "zod";
 import { wellsToRowsCols, indexToExcelCell } from "./utils";
-import { RowAnnotationGutter } from "./RowAnnotationGutter";
+import { RowAnnotationGutter, ColAnnotationGutter } from "./AnnotationGutters";
+import { AnnotationBar } from "./AnnotationBar";
+import { useState } from "react";
 
 const PlateSelectionSchema = z.object({
   wells: z.array(z.number()),
   className: z.string().optional(),
 });
 export type PlateSelection = z.infer<typeof PlateSelectionSchema>;
 
-export interface PlateProps {
+export interface WellAnnotation<T extends Record<string, unknown>> {
```

Comment:
These are implemented as interfaces extending Records as opposed to the zod types? It does feel like clutter to have these defined as top level types because really it seems like the application that uses this component is going to have their own types which they map back and forth into the components annotation format.

Reply:
That said, it feels like we might want to move these to a types file within the Plate folder.

Reply:
Let's chat about the purpose of these on our next call!

Moved into a ./types.ts file in 0ac54c4